test/integration/graphql/business.test.js

```diff
@@ -177,3 +177,49 @@ describe('Query.business.land', () => {
     })
   })
 })
+
+describe('Query.Business.cph', () => {
+  it('cph', async () => {
+    const result = await graphql({
+      source: `#graphql
+      query BusinessCPH {
+        business(id: "ID") {
+          cph {
+            number
+            parcelNumbers
+            parish
+            startDate
+            expiryDate
+            species
+            coordinate {
+              x
+              y
+            }
+          }
+        }
+      }
+      `,
+      schema,
+      contextValue: fakeContext
+    })
+
+    expect(result).toEqual({
+      data: {
+        business: {
+          cph: [{
+            number: expect.any(String),
```

Comment:
Can you match how the other integration tests are done so we are checking against actual values and not just types

Reply:
Please see latest commit, it's done now